app/javascript/src/components/editor/Compiler.svelte

```diff
@@ -41,17 +43,25 @@
 
 <svelte:window on:keydown={keydown} />
 
-<button class="button button--secondary button--square tooltip" on:click={doCompile}>
-  {#if compiling}
-    Compiling...
+<svelte:element
+  this={inline ? "span" : "button"}
+  class={inline ? "w-100" : "button button--secondary button--square tooltip"}
+  on:click={doCompile}
+>
+  {#if inline && copied}
+    Compiled and copied!
   {:else}
-    Compile
+    {#if compiling}
+      Compiling...
+    {:else}
+      Compile
+    {/if}
   {/if}
 
-  {#if copied}
+  {#if copied && !inline}
     <div transition:fly={{ y: 5, duration: 150 }} class="tooltip__content bg-primary text-pure-white block">
       Copied to clipboard
     </div>
   {/if}
-</button>
+</svelte:element>
```

Comment:
The reason I had to do the "Compiled and copied!" text was because the tooltip was inserted inside the dropdown. However, I think we could probably move this alert to be an actual alert (once #351 is merged).

I will look into the `.empty-button` class, thanks for the data!

Reply:
With this PR (https://github.com/Mitcheljager/workshop.codes/pull/356) merged you will no longer need to worry about that, it can simply be a button.

Reply:
Seems like it can simply be a button right now too. I noticed `.empty-button` changed the font family of the button, so I had to remove it. Instead, `button.dropdown__item` applies similar styles.